core/src/test/scala/integration/kafka/api/BaseAdminIntegrationTest.scala

```diff
@@ -230,6 +232,7 @@ abstract class BaseAdminIntegrationTest extends IntegrationTestHarness with Logg
     config.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers())
     config.put(AdminClientConfig.REQUEST_TIMEOUT_MS_CONFIG, "20000")
     config.put(AdminClientConfig.DEFAULT_API_TIMEOUT_MS_CONFIG, "40000")
+    config.put(AdminClientConfig.CLIENT_ID_CONFIG, clientId)
```

Comment:
This will impact all admins from this test. Maybe you can add it to `testCreatePartitionWithOptionRetryOnQuotaViolation` only

Reply:
ok, done